.gitignore

```diff
@@ -2,10 +2,6 @@
 # will have compiled files and executables
 target/
 
-# Remove Cargo.lock from gitignore if creating an executable, leave it for libraries
```

Comment:
Why? It's expected to ignore lock.

Reply:
Current GitHub Action workflow requires hashing `Cargo.lock` for caching. If this is intended, I'll configure it to cache for specific task instead of `Cargo.lock`.